GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs

```diff
@@ -117,6 +117,13 @@ public override string GetNamedPipeName(string enlistmentRoot)
             return POSIXPlatform.GetNamedPipeNameImplementation(enlistmentRoot);
         }
 
+        public override string GetGVFSServiceNamedPipeName()
+        {
+            // Pipes are stored as files on POSIX, use a rooted pipe name
+            // in the same location as the service to keep full control of the location of the file
+            return this.GetDataRootForGVFSComponent(GVFSConstants.Service.ServiceName) + ".pipe";

```

Comment:
Not your change, and looking for your opinion.  

I've had some trouble remembering what `GetDataRootForGVFSComponent` returns (and if it's a global path).

Do you think this should be renamed (perhaps to something like `GetGlobalDataRootForGVFSComponent`)?

It looks like in the git world "global" means per-user, and so that would seem to be consistent.

Reply:
> It looks like in the git world "global" means per-user, and so that would seem to be consistent.

I guess except on Windows `GetDataRootForGVFSComponent` returns something system wide ... maybe there is a better name.

Reply:
I wonder if `GetDataDirectoryForService`, `GetDataDirectoryForFeature` would be easier... but I am not sure... Also - what is global on Windows is not currently global on macOS...